tests/integration/crunchers/test_cruncher_lead_gas.py

```diff
@@ -106,18 +119,18 @@ def test_derive_relationship_averaging_info(self, test_db, extra_info):
         )
         variable_follower = "Emissions|HFC|C5F12"
         variable_leader = ["Emissions|HFC|C2F6"]
-        cruncher = tcruncher.derive_relationship(
-            variable_follower, variable_leader
-        )
+        cruncher = tcruncher.derive_relationship(variable_follower, variable_leader)
         lead_db = test_db.filter(variable=variable_leader)
         infilled = cruncher(lead_db)
         # In both cases, the average follower value at the latest time is 2. We divide
         # by the value in 2015, which we have data for in both cases.
         assert np.allclose(
             infilled.data["value"],
-            2 * lead_db.data["value"] / lead_db.data["value"].loc[
-                lead_db.data[lead_db.time_col] == max(lead_db.data[lead_db.time_col])
-            ].values
+            2
```

Comment:
really? I'm sure the algorithm spat this out, but why??

Reply:
god this is yuck. I think it's trying to:

a) start lines with binary operators (hence all the `.` and `*` etc. at start of lines
b) use brackets and parentheses rather than line continuations or long lines

more deets here https://stackoverflow.com/a/7942617

In such cases I tend to take black's ugly formatting as a sign that my code is doing too much so would re-write this to 

```python
lead_db_time = lead_db.data[lead_db.time_col]
        latest_time = lead_db_time == max(lead_db_time)
        expected = (
            2 * lead_db.data["value"] / lead_db.data["value"].loc[latest_time].values
        )
        assert np.allclose(infilled.data["value"], expected)
```

Reply:
I guess that's better.